# This message works with bools represented by integers to describe the different states
# of obstacles in certain directions. 0 means false --> no obstacle, 1 means true --> obstacle
# Fields:
# front_obstacle
# left_obstacle
# right_obstacle


int32 front_obstacle
int32 left_obstacle
int32 right_obstacle


